lib/redirect.js

```diff
@@ -150,6 +152,66 @@ Redirect.prototype.onResponse = function (response) {
         request.removeHeader('authorization')
       }
     }
+  } else if (request.formData &&
```

Comment:
Add:
1. TODOs to expose these errors and edge cases as events.
2. Boundary comments explaining that this is a Postman request specific change.
3. Updates to the fork differences list with the same (if not done already)